frame/balances/src/lib.rs

```diff
@@ -180,19 +180,19 @@ use frame_system::{self as system, ensure_signed, ensure_root};
 pub use self::imbalances::{PositiveImbalance, NegativeImbalance};
 
 pub trait WeightInfo {
-	fn transfer(u: u32, e: u32, ) -> Weight;
-	fn transfer_best_case(u: u32, e: u32, ) -> Weight;
-	fn transfer_keep_alive(u: u32, e: u32, ) -> Weight;
-	fn set_balance(u: u32, e: u32, ) -> Weight;
-	fn set_balance_killing(u: u32, e: u32, ) -> Weight;
+	fn transfer() -> Weight;
+	fn transfer_keep_alive() -> Weight;
+	fn set_balance_creating() -> Weight;
+	fn set_balance_killing() -> Weight;
```

Comment:
I see you went with exposing the details in the WeightInfo. Intentional design decision or temporary solution until we get to the final version?

Reply:
It may be that this changes, but for now, this is designed as intended. Once we introduce the `WeightInfo` attribute to the `decl_module` macro, then we will really know where this lands